molecule/shared/tests/test_default.rb

```diff
@@ -50,3 +50,21 @@
               params: { consumers: 1 }) do
   its('status') { should eq 204 }
 end
+
+# Ensure disk check exists
+describe file('/etc/sensu/conf.d/sensu_masters/check_disk_usage.json') do
+  it { should exist }
+  its('content') { should match(/check-disk-usage/) }
+end
+
+# Ensure disk metrics exists
+describe file('/etc/sensu/conf.d/sensu_checks/metrics_disk_usage.json') do
+  it { should exist }
+  its('content') { should match(/metrics-disk-usage/) }
+  its('content') { should match(/60/) }
```

Comment:
I think this is a good start, but, since there is that Jinja2 template in there, what if we switch this to using https://www.inspec.io/docs/reference/resources/json/ and validate that `interval` equals `60`. 

i.e.:
```ruby
describe json('/etc/sensu/conf.d/sensu_checks/metrics_disk_usage.json'') do
  its(['checks','metrics_disk_usage','command') { should eq 'metrics-disk-usage.rb' }
  its(['checks','metrics_disk_usage','command']) { should eq 60 }
end
```

I think that'll also get us json validation as well incase something breaks this templating in the future.